### PR TITLE
crypto++: version bump to 8.2

### PR DESCRIPTION
**Pull request by kikadf**

### CODE REVIEW

File: dev-libs/crypto++/crypto++-8.2.0.recipe [hunk 1]
```diff
@@ -86,8 +84,7 @@ Version: $portVersion
 Libs: -lcryptopp
 EOF
 
-	packageEntries devel \
-		$relativeDevelopDir
+	packageEntries devel $relativeDevelopDir
```
Comment (Begasus) [inline dev-libs/crypto++/crypto++-8.2.0.recipe]: Is $reletiveDevelopDir needed here? Or could you just use $developDir?

Comment (kikadf) [reply dev-libs/crypto++/crypto++-8.2.0.recipe]: Well, I don't know what is the differences between $relativeDevelopDir
and $developDir.
I tried build without this line and I got error, so just leave there.
Can you help me, where can I found this descriptions? Should I check the haikuporter source?

Comment (kikadf) [reply dev-libs/crypto++/crypto++-8.2.0.recipe]: Ok, I found in the haikuporter's wiki. Absolut path vs relative path.
So:
developDir=/boot/system/develop
relativeDevelopDir=/develop ?

Comment (Begasus) [reply dev-libs/crypto++/crypto++-8.2.0.recipe]: From the wiki "$relativeDevelopDir: $developDir as a relative directory (to prefix)." https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes, haven't checked if it works with $developDir :)

Comment (waddlesplash) [reply dev-libs/crypto++/crypto++-8.2.0.recipe]: Most invocations of `packageEntries` use $developDir, so try that.

Comment (kikadf) [reply dev-libs/crypto++/crypto++-8.2.0.recipe]: Done.

**Comment by Begasus** [comment]
Haven't checked but LGTM

**Comment by kikadf** [comment]
Please wait for merge, I found the patch isn't complete. I am fixing that, then upgrade the PR.

**Comment by kikadf** [comment]
The patchset is fixed, build and works fine on x86_64. Only found one package yet, what has cryptopp dependency: codecrypt (Tested and build fine on x86_64).

**Comment by waddlesplash** [comment]
Very nice, thanks! Sorry for the delay!

**Pull request merged**